client/my-sites/site-settings/cloudflare.js

```diff
@@ -30,32 +40,74 @@ const Cloudflare = () => {
 		<>
 			{ showCloudflare && (
 				<>
-					<SettingsSectionHeader title={ translate( 'Cloudflare CDN' ) } />
-					<Card>
+					{ ! sitePlan && <QuerySitePlans siteId={ siteId } /> }
+					<SettingsSectionHeader title={ translate( 'CDN' ) } />
+					<CompactCard>
 						<div className="site-settings__cloudflare">
+							<div className="site-settings__cloudflare-illustration">
+								<img src={ jetpackIllustration } alt="" />
+							</div>
 							<div className="site-settings__cloudflare-text">
+								<p className="site-settings__cloudflare-title">Jetpack Site Accelerator</p>
+								<p>{ translate( 'Comes built-in with WordPress.com Business plans.' ) }</p>
 								<p>
-									{ translate(
-										'Use the Cloudflare global server network to optimize your site content and create a faster experience for your users regardless of their device or location.'
-									) }
+									<a
+										onClick={ recordClick }
+										href="https://jetpack.com/features/design/content-delivery-network/"
+										target="_blank"
+										rel="noreferrer"
+									>
+										{ translate( 'Learn more' ) }
+									</a>
+								</p>
+							</div>
+						</div>
+					</CompactCard>
+					{ sitePlan && showBizUpsell && (
+						<UpsellNudge
+							title={ translate( 'Available on Business plan or higher' ) }
+							href={ upgradeLink }
+							event={ 'calypso_settings_cloudflare_cdn_upsell_nudge' }
+							showIcon={ true }
+							forceDisplay={ true }
```

Comment:
I'm not sure you need to set `forceDisplay` here...IIRC it was only needed for some cases in the sidebar upsell nudges. I could be wrong, tho. :)